packages/visualizations/src/components/Table/types.ts

```diff
@@ -8,9 +8,17 @@ export type DataFormat = typeof DATA_FORMAT[DataFormatKey];
 
 export type TableData = DataFrame;
 
+export const ColumnSort = {
+    asc: 'ASC',
+    desc: 'DESC',
+} as const;
+export type ColumnSortValues = typeof ColumnSort[keyof typeof ColumnSort];
+
 type BaseColumn = {
     key: string;
     title: string;
+    sorted?: ColumnSortValues;
```

Comment:
I would add a comment to explain what kind of value is expected. 

As `ColumnSortValues` is computed from other types, it's might not be easy to understand what is expected

Reply:
Intellisense gives the right possible values, no? We never really committed to jsdoc or anything for TS files as far as I know and pretty much rely on intellisense. I can add an obvious comment 🤷‍♂️

Reply:
Indeed Intellisense gives the values. I think I just hovered sorted during my testing.
I can confirm that with Intellisense `ColumnSortValues` we got the values